modules/ROOT/pages/gsn/what-is-the-gsn.adoc

```diff
@@ -0,0 +1,68 @@
+= What is the Gas Station Network?
+
+*https://gsn.openzeppelin.com[Website]*
```

Comment:
The link by itself looks sort of weird on the docsite, perhaps we could put it on the title?